generated/nidmm/nidmm.proto

```diff
@@ -346,6 +362,13 @@ enum MeasurementCompleteDest {
   MEASUREMENT_COMPLETE_DEST_NIDMM_VAL_LBR_TRIG0 = 1003;
 }
 
+enum MeasurementDestinationSlope {
```

Comment:
You could combine SampleTrigSlope and TriggerSlope with this and have just one enum that contains VAL_POSITIVE and VAL_NEGATIVE

Reply:
combined them